jdaviz/tests/test_utils.py

```diff
@@ -1,20 +1,80 @@
 import os
+import pytest
 import warnings
 
 import photutils
-import pytest
 from asdf.exceptions import AsdfWarning
+from astropy import units as u
 from astropy.utils import minversion
 from astropy.wcs import FITSFixedWarning
+from numpy.testing import assert_allclose
+from specutils import Spectrum1D
+
 from jdaviz import utils

```

Comment:
I don't think this import is needed anymore. Any remaining `utils.function_name` calls in this module should be changed to `function_name` only.
```suggestion
```